themes/default/content/registry/packages/aws-native/_index.md

```diff
@@ -250,3 +250,144 @@ resources:
 {{% /choosable %}}
 
 {{% /chooser %}}
+
+## Third Party Resources
+
+The SDK for the AWS Native Cloud Control provider only includes resources for which Amazon have published the corresponding CloudFormation specifications. AWS also supports [CloudFormation extensions via the CloudFormation registry](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html) which can also be accessed via Cloud Control.
+
+If you want to manage resources using Pulumi's AWS Native Cloud Control Provider which aren't in the SDK, you can use the [ExtensionResource](https://www.pulumi.com/registry/packages/aws-native/api-docs/extensionresource/) resource within root of the SDK. The input properties and output properties are untyped allowing passing in any arbitrary values which will be passed directly to the Cloud Control API, but utilising Pulumi's managed state model.
```

Comment:
nit. I'm not sure the "but utilising Pulumi's managed state model" clause at the end is super clear. Maybe "and tracked with Pulumi's managed state model"? Not totally sure what would read best.